magellan-library/src/main/java/com/wealthfront/magellan/Navigator.java

```diff
@@ -548,8 +548,18 @@ private void navigate(final Direction direction, final NavigationType navType, f
   private void animateAndRemove(
       final View from, final View to, final NavigationType navType, final Direction direction) {
     ghostView = from;
-    final Transition transitionToUse = overridingTransition != null ? overridingTransition : transition;
-    overridingTransition = null;
+    Transition currentTransition = transition;
+    if (overridingTransition != null) {
+      currentTransition = overridingTransition;
+      overridingTransition = null;
+    } else if (direction.equals(Direction.BACKWARD) && currentScreen().getLeaveTransition() != null) {
+      currentTransition = currentScreen().getLeaveTransition();
```

Comment:
I want to test this logic, but triggering the `whenMeasured` block is tough. I've tried manually calling `measure` and `layout`, but it's still not triggering it. Thoughts/suggestions?